explorer/interpreter/interpreter.cpp

```diff
@@ -1503,6 +1503,72 @@ auto Interpreter::StepStmt() -> ErrorOr<Success> {
         }
       }
     }
+    case StatementKind::For: {
+      static int CURRENT_INDEX_POS = 1;
+      static int TARGET_VAR_POS = 0;
+      static int END_INDEX_POS = 2;
```

Comment:
The `_pos` suffix on these names could be confusing, because they don't correspond to `act.pos()`. Maybe something like `_RESULT` instead?

Also,
- these should probably be `constexpr`
- we only use all-caps for macros. Constants should [use CamelCase](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/cpp_style_guide.md#general-naming-rules).